easybuild/easyconfigs/b/beagle-lib/beagle-lib-2.1.2-goolf-1.7.20.eb

```diff
@@ -2,6 +2,7 @@ easyblock = 'ConfigureMake'
 
 name = 'beagle-lib'
 version = '2.1.2'
+versionsuffix = '-Java-%(javaver)s'
```

Comment:
and this `beagle-lib` is used as a dep for `Beast`?

I would add a new `beagle-lib` easyconfig (with a version suffix & using Java 8), and change the dep for Beast, rather than changing an existing `beagle-lib` easyconfig...